compiler/src/parsing/parser.dyp

```diff
@@ -523,7 +523,7 @@ lam_args :
   | patterns? { Option.value ~default:[] $1 }
 
 lam_expr :
-  | lparen lam_args rparen thickarrow block_or_expr { Exp.lambda ~loc:(symbol_rloc dyp) $2 $5 }
+  | lparen lam_args comma? rparen thickarrow block_or_expr { Exp.lambda ~loc:(symbol_rloc dyp) $2 $5 }
```

Comment:
Looks like this is why the CI is failing.
```suggestion
  | lparen lam_args comma? rparen thickarrow block_or_expr { Exp.lambda ~loc:(symbol_rloc dyp) $2 $6 }
```

Reply:
Oops! 🙈 Let me fix that.

Reply:
I went ahead and committed it!